pkg/kademlia/kademlia_test.go

```diff
@@ -914,19 +916,38 @@ func TestKademlia_SubscribePeersChange(t *testing.T) {
 	})
 }
 
-func TestMarshal(t *testing.T) {
-	_, kad, ab, _, signer := newTestKademlia(nil, nil, kademlia.Options{})
+func TestSnapshot(t *testing.T) {
+	sa, kad, ab, _, signer := newTestKademlia(nil, nil, kademlia.Options{})
 	if err := kad.Start(context.Background()); err != nil {
 		t.Fatal(err)
 	}
 	defer kad.Close()
 
 	a := test.RandomAddress()
 	addOne(t, signer, kad, ab, a)
-	_, err := kad.MarshalJSON()
-	if err != nil {
-		t.Fatal(err)
+
+	snap := kad.Snapshot()
+
+	if snap.Connected != 0 {
+		t.Error("wrong connected value", snap.Connected)
```

Comment:
we try to stick to the convention of:
`t.Errorf/Fatalf("expected %d connected peers but got %d",snap.Connected, 0)`

Reply:
fixed